### PR TITLE
feat: Extend collection of usage metrics during license renewal (no-changelog)

### PR DESCRIPTION
**Pull request by csuermann**
## Summary
Submit a few more usage metrics to during license renewal, namely
- `totalWorkflows`
- `enabledUsers`
- `totalCredentials`
- `productionExecutions`
- `manualExecutions`

## Review / Merge checklist
- [x] PR title and summary are descriptive. **Remember, the title automatically goes into the changelog. Use `(no-changelog)` otherwise.** ([conventions](https://github.com/n8n-io/n8n/blob/master/.github/pull_request_title_conventions.md))
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included.
   > A bug is not considered fixed, unless a test is added to prevent it from happening again.
   > A feature is not complete without tests.

### CODE REVIEW

File: packages/cli/src/License.ts [hunk 1]
```diff
@@ -94,11 +100,22 @@ export class License {
 	}
 
 	async collectUsageMetrics() {
```
Comment (netroy) [inline packages/cli/src/License.ts:102]: instead of keeping adding unrelated code to `License`, we should extract this method into a separate `UsageMetricsService`.

**Review by netroy — DISMISSED** [review]

File: packages/cli/src/databases/repositories/usageMetrics.repository.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+			},
+		] = (await this.query(`
+			SELECT
+				(SELECT COUNT(*) FROM ${this.fullPrefix}user WHERE disabled = false) AS enabled_user_count,
+				(SELECT COUNT(*) FROM ${this.fullPrefix}workflow_entity WHERE active = true) AS active_workflow_count,
+				(SELECT COUNT(*) FROM ${this.fullPrefix}workflow_entity) AS total_workflow_count,
+				(SELECT COUNT(*) FROM ${this.fullPrefix}credentials_entity) AS total_credentials_count,
+				(SELECT SUM(count) FROM ${this.fullPrefix}workflow_statistics WHERE name IN ('production_success', 'production_error')) AS production_executions_count,
+				(SELECT SUM(count) FROM ${this.fullPrefix}workflow_statistics WHERE name IN ('manual_success', 'manual_error')) AS manual_executions_count;
+		`)) as Row[];
```
Comment (krynble) [inline packages/cli/src/databases/repositories/usageMetrics.repository.ts:51]: Screenshot 2024-01-19 at 10 42 49
Queries are failing for Postgres

Comment (ivov) [reply packages/cli/src/databases/repositories/usageMetrics.repository.ts]: Thank you, please see [46d27ec](https://github.com/n8n-io/n8n/pull/8369/commits/46d27ece0443c1e07c3619222f8f6505f2617b14)

Comment (ivov) [reply packages/cli/src/databases/repositories/usageMetrics.repository.ts]: Should be fixed now, thank you

**Review by krynble — APPROVED** [review]
LGTM

**Comment by ivov** [comment]
@netroy When you can please approve, it's blocking CI

Capture 2024-01-22 at 09 28 08@2x
